Code/GraphMol/MolOps.h

```diff
@@ -390,6 +390,9 @@ struct RDKIT_GRAPHMOL_EXPORT AdjustQueryParameters {
       false; /**<  sets non-ring single bonds between two aromatic or conjugated
                 atoms to SINGLE|AROMATIC */
 
+  bool setGenericQueryFromProperties =
```

Comment:
We discussed and rejected a change like this in #6332 
I don't want to add a circular dependency from GraphMol<->GenericGroups.

To avoid this, @bjonnh-work added `adjustQueryPropertiesWithGenericGroups()` in `GenericGroups`.

Reply:
I have removed that dependency, and I also removed that entire test that generated it.  That test is already in a GenericGroups test and uses the adjustQueryPropertiesWithGenericGroups that is referenced above.